design/tracing-design.md

```diff
@@ -0,0 +1,170 @@
+# Design for Supporting Tracing in Contour
+
+Status: Draft
+
+## Abstract
+Envoy has rich support for [distributed tracing][1].
+However, Contour does not currently offer a way to configure Envoy for tracing.Exporting trace data to OpenTelemetry has been supported since envoy v1.23. This design document provides the implementation of exporting tracing data to [OpenTelemetry][2] in Contour.
```

Comment:
```suggestion
Exporting trace data to OpenTelemetry has been supported since envoy v1.23.
However, Contour does not currently offer a way to configure Envoy for tracing.
This design document provides the implementation of exporting tracing data to [OpenTelemetry][2] in Contour.
```